src/components/CCPDashboard/PatientDetailsDialog.tsx

```diff
@@ -34,6 +50,75 @@ interface PatientDetail {
   styles?: object;
 }
 
+export const PatientDetailsDialogRouting = ({
+  match: {
+    params: { eventId, patientId, ccpId },
+  },
+}: {
+  match: {
+    params: {
+      eventId: string;
+      patientId?: string;
+      ccpId: string;
+    };
+  };
+}) => {
+  const [openDetails, setOpenDetails] = React.useState(true);
+  const history = useHistory();
+  const classes = useStyles();
+
+  const { data } = useQuery(
+    patientId ? GET_PATIENT_BY_ID(patientId) : GET_ALL_PATIENTS
+  );
+
+  const patient: Patient = data ? data.patient : undefined;
```

Comment:
nit: usually makes more sense to set objects to `null` if you want it to be falsy
```suggestion
  const patient: Patient = data ? data.patient : null;
```